soh/src/code/z_sram.c

```diff
@@ -790,6 +790,9 @@ void Sram_InitSave(FileChooseContext* fileChooseCtx) {
             gSaveContext.eventChkInf[1] |= (1 << 3);
             gSaveContext.eventChkInf[1] |= (1 << 4);
 
+            // Got Zelda's Letter
+            gSaveContext.eventChkInf[4] |= 1;
```

Comment:
we're already doing this later with `// make sure saria is at SFM` (probably worth just updating that comment to explain that it's also the got zelda's letter flag. i'd probably change it to
```cpp
// Set "Got Zelda's Letter" flag (also ensures saria is at SFM and TODO: what else does this flag do?)
```

Reply:
I totally missed that. I'll change this.